android/src/androidTest/AndroidManifest.xml

```diff
@@ -11,9 +11,9 @@
     <!-- Gradle generates an <instrumentation> block during the build process,
          but in order to work with other build systems, that don't do that,
          there must be such a block declared here. -->
-    <instrumentation
-        android:name="android.test.InstrumentationTestRunner"
-        android:targetPackage="com.android.i18n.addressinput.test" />
+    <!--<instrumentation-->
```

Comment:
This does not look good. Either this <instrumentation> block is no longer necessary, in which case it should be deleted together with the comment explaining its purpose and the commit message updated to explain why it's no longer necessary, or else it should not be commented out.